types/arangodb/index.d.ts

```diff
@@ -951,8 +952,9 @@ declare namespace ArangoDB {
 
         // AQL
         _createStatement(query: Query | string): Statement;
+        _query(query: Query, options?: QueryOptions): Cursor;
         _query(
-            query: Query | string,
+            query: string,
```

Comment:
Note: the old typings were actually incorrect: `db._query` wouldn't take any extra arguments if the first argument is a `Query` object. ArangoDB 3.5 adds support for passing a `Query` object with `QueryOptions`.